consensus/ibft/snapshot.go

```diff
@@ -366,21 +366,29 @@ func newSnapshotStore() *snapshotStore {
 }
 
 // loadFromPath loads a saved snapshot store from the specified file system path
-func (s *snapshotStore) loadFromPath(path string) error {
+func (s *snapshotStore) loadFromPath(i *Ibft) error {
 	// Load metadata
 	var meta *snapshotMetadata
-	if err := readDataStore(filepath.Join(path, "metadata"), &meta); err != nil {
-		return err
-	}
+	if err := readDataStore(filepath.Join(i.config.Path, "metadata"), &meta); err != nil {
+		// if we can't read file metadata file delete it and sync again
+		// and logout that we've had some problems
+		i.logger.Error("could not read metadata file","err",err.Error())
+		i.logger.Error("removing faulty metadata file...")
```

Comment:
This log should happen after you run `os.Remove` and it should go `Removed invalid snapshot store file`

Reply:
Fixed with commit b0bf7cd004339996424fc7301b460a33873b367f